src/screens/Menu/RecoverySettings.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { withTheme } from 'styled-components/native';
+import type { NavigationScreenProp } from 'react-navigation';
+import ContainerWithHeader from 'components/Layout/ContainerWithHeader';
+import { ScrollWrapper } from 'components/Layout';
+import { spacing } from 'utils/variables';
+import { getThemeColors } from 'utils/themes';
+import { BACKUP_WALLET_IN_SETTINGS_FLOW, REVEAL_BACKUP_PHRASE } from 'constants/navigationConstants';
+
+import type { RootReducerState } from 'reducers/rootReducer';
+import type { BackupStatus } from 'reducers/walletReducer';
+import type { Theme } from 'models/Theme';
+
+import { SettingsSection } from './SettingsSection';
+
+type Props = {
+  navigation: NavigationScreenProp<*>,
+  backupStatus: BackupStatus,
+  theme: Theme,
+};
+
+class RecoverySettings extends React.Component<Props> {
+  getGlobalSection = () => {
+    return [
+      {
+        key: 'linkedDevices',
+        title: 'Linked devices',
+        subtitle: 'Assign your contacts as recovery agents for restoring the wallet',
+        disabled: true,
+        label: 'soon',
+      },
+    ];
+  }
+
+  getKeyWalletSection = () => {
+    const { backupStatus, theme, navigation } = this.props;
+    const isBackedUp = backupStatus.isImported || backupStatus.isBackedUp;
+    const colors = getThemeColors(theme);
+    return [
+      {
+        key: 'view12Words',
+        title: 'View 12 words',
+        onPress: () => navigation.navigate(REVEAL_BACKUP_PHRASE),
+        hidden: !isBackedUp,
+      },
+      {
+        key: 'viewPrivateKey',
+        title: 'View private key',
+        onPress: () => navigation.navigate(REVEAL_BACKUP_PHRASE, { showPrivateKey: true }),
```

Comment:
In REVEAL_BACKUP_PHRASE, it would be great to update header is PK is visible. It now says "backup phrase".